### PR TITLE
Add ADOPTERS.md file

### PR DESCRIPTION
**Pull request by alban**
This file will display users of Inspektor Gadget.

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+This page contains a non-comprehensive list of [Inspektor Gadget](https://inspektor-gadget.io/) users that we know of.
+
+## Adding your org to this list
+
+If you would like your organization or project to be added to this list, please open a PR with your name, link, and a brief description of your use case.
+
+## Inspektor Gadget adopters (in alphabetical order):
+
+| Adopter  | Description |
```
Comment (blanquicet) [inline ADOPTERS.md:11]: What about https://github.com/microsoft/retina?

Comment (alban) [reply ADOPTERS.md:11]: https://retina.sh/docs/Metrics/plugins/Linux/dns#architecture

Comment (blanquicet) [reply ADOPTERS.md:11]: Also the [tcpretrans](https://retina.sh/docs/Metrics/plugins/Linux/tcpretrans) plugin uses [Inspektor Gadget](https://github.com/microsoft/retina/blob/e4d2d3e0e8370b6f21992863f489ceefcb1fa3ee/pkg/plugin/tcpretrans/tcpretrans_linux.go#L4).

**Review by blanquicet — APPROVED** [review]
LGTM!

**Comment by alban** [comment]
Thanks. The tests still fail but it is not related to this change, so I will merge.

**Pull request merged**